setup.cfg

```diff
@@ -11,18 +11,6 @@ ignore = D202,D203,W503,E203
 max-line-length = 79
 exclude = versioneer.py
 
-[pydocstyle]
-inherit = false
-convention = numpy
-
-[isort]
-known_third_party = cached_property,click,clickhouse_driver,dateutil,google,graphviz,impala,kudu,mock,multipledispatch,numpy,pandas,pkg_resources,plumbum,psycopg2,pyarrow,pydata_google_auth,pygit2,pymapd,pymysql,pyspark,pytest,pytz,regex,requests,setuptools,sphinx_rtd_theme,sqlalchemy,thrift,toolz
```

Comment:
Were not these required previously?

Reply:
Nope. I'm not sure why they were there. I think these are from a very long time ago.